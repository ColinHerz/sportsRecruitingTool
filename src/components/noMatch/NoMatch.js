import { Link } from "react-router-dom";
import React from "react";

import "./noMatch.scss";

const NoMatch = props => {
	return (
		<main id="not-found">
			<h2>404</h2>

			<h3>Page Not Found</h3>

			<p>The Page you are looking for doesn&apos;t exist or an error occured. Go <Link to="/">home</Link>.</p>

			<p>Or panic. I&apos;m not your dad.</p>
		</main>
	);
};

export default NoMatch;
