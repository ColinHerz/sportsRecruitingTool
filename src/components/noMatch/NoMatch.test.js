/* eslint-env jest */

// MemoryRouter needed because header contains NavLinks, which need to
// be rendered within a Router
import { MemoryRouter } from "react-router-dom";
import React from "react";
import renderer from "react-test-renderer";

import NoMatch from "./NoMatch.js";

describe(`No match component`, () => {
	beforeEach(() => {
		jest.resetModules();
	});

	it(`renders correctly`, () => {
		const props = {};

		const component = renderer.create(
			<MemoryRouter>
				<NoMatch {...props} />
			</MemoryRouter>
		);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
