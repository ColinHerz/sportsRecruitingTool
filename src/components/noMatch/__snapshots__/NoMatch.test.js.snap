// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No match component renders correctly 1`] = `
<main
  id="not-found"
>
  <h2>
    404
  </h2>
  <h3>
    Page Not Found
  </h3>
  <p>
    The Page you are looking for doesn't exist or an error occured. Go 
    <a
      href="/"
      onClick={[Function]}
    >
      home
    </a>
    .
  </p>
  <p>
    Or panic. I'm not your dad.
  </p>
</main>
`;
